src/utils/errorCategories.ts

```diff
@@ -0,0 +1,12 @@
+export enum ErrorCategory {
+  BudgetError = 'Budget Error',
```

Comment:
This is just an idea...
I'd remove the `Error` word from all.

Reply:
Like this idea